### PR TITLE
Feature/fix up version logic

### PR DESCRIPTION
**Pull request by mnazbro**

### CODE REVIEW

**Comment by mnazbro** [comment]
@punya - Could I get a CR on this?

File: README.md [hunk 1]
```diff
@@ -39,3 +42,12 @@ License
...
 Gradle Miniconda Plugin is released by Palantir Technologies, Inc. under the Apache 2.0 License. See the included
 [LICENSE](LICENSE) file for details.
+
+Backwards Compatibility Breaks
+------------------------------
+
+### 0.4.0
+The bootstrap Python is now placed in `bootstrapDirectoryPrefix/minicondaVersion`. Users now must set
+`bootstrapDirectoryPrefix` instead of `bootstrapDirectory`. You can still get the new directory of the bootstrap Python
+by calling the `bootstrapDirectory` property.
```
Comment (punya) [inline README.md]: by referring to

**Comment by punya** [comment]
Tiny documentation fix, but otherwise this looks good.

**Comment by mnazbro** [comment]
@punya I've addressed the comments. I'll merge this if there are no comments tomorrow.

**Comment by punya** [comment]
@mnazbro is there anything else you'd like before we cut a release?

**Comment by mnazbro** [comment]
Nope, this should be good!

**Pull request merged**